### PR TITLE
Features/form icons

### PR DESCRIPTION
**Pull request by casaper**
Not Some form fields need icons, but maybe someone has a good idea what icon is suitable for mentor/interests?

### CODE REVIEW

File: app/views/events/timeline.html.haml [hunk 1]
```diff
@@ -29,27 +29,23 @@
...
+                = event.end_at.to_s(:time)
+              %span.date
+                = event.end_at.strftime("%d.%m")
           .card-action
             - if can?(:edit, event) && event.persisted?
               %a{ href: url_for([:edit, event])} Edit event
             .avatars
               - event.attendances.each_with_index do |a, i|
-                = image_tag a.guest.avatar_url, class: ['circle', "attendance-state-#{a.state}"], style: "right: #{i*40}px"
+                = image_tag a.guest.avatar_url, class: 'circle', style: "right: #{i*40}px"
```
Comment (huerlisi) [inline app/views/events/timeline.html.haml]: I think this is a merge error, the `attendance-state-` class should still be present.

**Comment by huerlisi** [comment]
Added a small comment about the timeline changes.

And you should switch the commits to use these conventions: https://github.com/bcoe/conventional-changelog-standard/blob/master/convention.md

You may change the commit messages by using `git rebase -i origin/master`. Instructions should appear. Then you can `git push --force` to update this PR.

Thanks!

**Comment by huerlisi** [comment]
This looks good, but some tests seem to fail.... Need to investigate that.

**Comment by casaper** [comment]
> @huerlisi: but some tests seem to fail.... Need to investigate that.

locally a ```rake``` is running smoothly returning a 
```
Finished in 12.2 seconds (files took 3.3 seconds to load)
68 examples, 0 failures
```

And the jenkins test seems to fail with some phantomjs stuff. I'm not 100% sure, but I guess the failures are not related to my mods here. Are they?

**Comment by huerlisi** [comment]
Yepp, it's not your fault. The download of the phantomjs seems not to work due to bitbucket getting too many requests. I've triggered a rebuild and it's all green now:-)

**Comment by huerlisi** [comment]
Good work, thanks for all this!

@casaper I've done some rebasing to get the history nice and clean;-) I can show you some tricks in this regard.

The commit messages were already close to what we expect. A small thing about the "topic": they are used to group the changes in the changelog. So it makes sense to keep the number of topics low. I often quickly look at the current git history to figure out a sensible "topic". Most of the time this could be the (pluralized) name of the affected model, or some general ones like "design" or "ux".

**Comment by casaper** [comment]
@huerlisi Thanks allot. I'm aware that my git handling needs some improovement, and of course I'll be grateful for beeing tought to do better.

**Comment by huerlisi** [comment]
@casaper very happy about you stuff! Good progress!

I'll just be noisy so you can improve even further :sunglasses:

**Pull request merged**